documentation/spec/src/interoperability/ethereum-bridge.md

```diff
@@ -49,21 +48,26 @@ pub enum EthereumEvent {
 }
 ```
 
-Each event should have a list of the validators that have seen
-this event and the current amount of stake associated with it. This
-will need to be appropriately adjusted across epoch boundaries. However,
-once an event has been seen by 2/3 of the voting power, it is locked into a
-`seen` state. Thus, even if after an epoch that event has no longer been
-reported as seen by 2/3 of the new staking validators voting power, it is still
-considered as `seen`.
-
-Each event from Ethereum should include the minimum number of confirmations
-necessary to be considered seen. Validators should not vote to include events
-that have not met the required number of confirmations. Furthermore, validators
-should not look at events that have not reached protocol specified minimum
-number of confirmations (regardless of what is specified in an event). This 
-constant may be changeable via governance. Voting on unconfirmed events is
-considered a slashable offence.
+Each event will be stored with a list of the validators that have ever seen it 
+as well as the fraction of total voting power that has ever seen it. 
+Once an event has been seen by 2/3 of voting power, it is locked into a
+`seen` state, and acted upon.
+
+There is no adjustment across epoch boundaries - e.g. if an event is seen by 1/3
+of voting power in epoch n, then seen by a different 1/3 of voting power in 
+epoch m>n, the event will be considered `seen` in total. Validators may never
+vote more than once for a given event.
+
+### Minimum confirmations
+There will be a protocol-specified minimum number of confirmations that events
+must reach on the Ethereum chain, before validators should vote to include them
+on Namada. Validators should not vote to include events that have not met this 
+required number of confirmations. This constant will be changeable via 
+governance. Voting on unconfirmed events is considered a slashable offence.
+
+In a future version of the Ethereum bridge, we may allow an additional per-event
+minimum number of confirmations that an event should have to reach before being
+included in Namada.
```

Comment:
Currently our Ethereum events don't include a custom additional minimum number of confirmations, I think we should do this as an extension once we have protocol-level minimum number of confirmations working